testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/microprofile/metrics/MetricsHelper.java

```diff
@@ -0,0 +1,98 @@
+package org.jboss.as.test.integration.ws.microprofile.metrics;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import java.io.StringReader;
+
+import javax.json.Json;
+import javax.json.JsonNumber;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.junit.Assert;
+
+public class MetricsHelper {
```

Comment:
That whole helper class is a copy/paste of org.wildfly.test.integration.microprofile.metrics.MetricsHelper.

Instead of this duplication, you should move the class to a module common to the `basic` and `ws` test suite (e.g. `wildfly-testsuite-shared`)

Reply:
OK. I moved this to wildfly-testsuite-shared module.